src/test/java/org/folio/rest/impl/AccountsRefundAPITests.java

```diff
@@ -125,17 +150,17 @@ public void fullRefundOfClosedAccountWithTransfer() {
 
   @Test
   public void fullRefundOfClosedAccountWithPaymentAndTransfer() {
-    double initialAmount = 6.0;
-    double payAmount = 3.0;
-    double transferAmount = 2.0;
-    double waiveAmount = 1.0;
-    double refundAmount = 5.0;
+    MonetaryValue initialAmount = new MonetaryValue(6.0);
+    MonetaryValue payAmount = new MonetaryValue(3.0);
+    MonetaryValue transferAmount = new MonetaryValue(2.0);
+    MonetaryValue waiveAmount = new MonetaryValue(1.0);
+    MonetaryValue refundAmount = new MonetaryValue(5.0);
 
     List<Matcher<JsonObject>> expectedFeeFineActions = Arrays.asList(
-      feeFineActionMatcher(FIRST_ACCOUNT_ID, -3.0, payAmount, CREDIT.getFullResult(), REFUND_TO_PATRON),
-      feeFineActionMatcher(FIRST_ACCOUNT_ID, -5.0, transferAmount, CREDIT.getFullResult(), REFUND_TO_BURSAR),
-      feeFineActionMatcher(FIRST_ACCOUNT_ID, -2.0, payAmount, REFUND.getFullResult(), REFUNDED_TO_PATRON),
-      feeFineActionMatcher(FIRST_ACCOUNT_ID,  0.0, transferAmount, REFUND.getFullResult(), REFUNDED_TO_BURSAR)
+      feeFineActionMatcher(FIRST_ACCOUNT_ID, BALANCE_MINUS[3], payAmount, CREDIT.getFullResult(), REFUND_TO_PATRON),
+      feeFineActionMatcher(FIRST_ACCOUNT_ID, BALANCE_MINUS[5], transferAmount, CREDIT.getFullResult(), REFUND_TO_BURSAR),
+      feeFineActionMatcher(FIRST_ACCOUNT_ID, BALANCE_MINUS[2], payAmount, REFUND.getFullResult(), REFUNDED_TO_PATRON),
```

Comment:
Again, if these values are just "reverted" payment, refund, transfer amount etc., maybe we can use the existing variables instead of creating new ones?